src/components/Header/index.jsx

```diff
@@ -24,12 +24,14 @@ export default function Header() {
                     <Link to="https://conda.org/">conda</Link>.
                 </h1>
                 <div className={styles.header_content_input}>
-                    <Link
-                        to="/docs/"
-                        className="button button--secondary button--lg homepage_button"
-                    >
+                    <Link to="/docs/"
+                        className="button button--secondary button--lg homepage_button">
                         Explore conda-forge
                     </Link>
+                    <Link to="/download/"

```

Comment:
What if we use `/miniforge/` here? I assume this is superseding https://conda-forge.org/miniforge

Reply:
If I read "Download" on a website for a repository of packages, I don't know if I'd expect to find an installer for _one_ of the package managers that are able to consume those packages. So if we want to make this a Download page, it should inform about Miniforge, micromamba, and pixi (so far).

Reply:
The download page is where you find stuff to download. There could be more things than the installer eventually.

Also, "Miniforge" as a topbar item is not something someone new to the project will understand.